code/game/objects/items/stacks/medical.dm

```diff
@@ -109,10 +53,29 @@
 	icon_state = "brutepack"
 	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
-	heal_brute = 40
+	var/heal_brute = 40

```

Comment:
i think its better to have the var/ at the parent and change in the childs, so if someone wants to make bandages slighty heal you they can or make bruise packs that heal burn or brute

Reply:
There is no way someone is making bruise packs that heal both brute and burn, because that would be conceptually harebrained.

Nobody did it in the last few years and nobody will do it now. Having those variable in every single medical stack, even though they never get used is just nonsensical.

If you were saying something like "I want to add a medical stack that heals both brute and burn", I would consider it, but I don't think you or anyone is actually stupid enough to do that. And even if, you could just add a new stack that does that and add the variable there.

Reply:
alien glue

Reply:
in defense of tlat (lol) amount healed should probably be a consistent var.

would be nice if there was a second var that selected which damage to heal (gauze is an exception to both and I haven't bothered to check if it actually needs to be subtyped to medical).